app/src/main/java/com/rafraph/pnineyHalachaHashalem/myAudio.java

```diff
@@ -65,19 +89,24 @@
     ArrayList<String> sections;
     private String url;
     View view;
-
+    Elements headers;
+    Document doc = null;
+    private Spinner spinner;
     private final Handler handler = new Handler();
-    private TextView header_text;
+
     private String header;
     public ListView listview;
     public String book_name;
+    public TextView txtI;
     Bundle extras;
     private MediaPlayerService playerService;
     boolean serviceBound = false;
     boolean firstCall;
     private Intent playerIntent;
     boolean clickOnItemFromList = false;
-
+    public Bundle b=null;

```

Comment:
Pleave rename **b** to some other name that will be easy to understand the role of this variable

Reply:
מחקתי את המשתנה הזה היה מיותר